core/app/helpers/spree/base_helper.rb

```diff
@@ -214,6 +214,64 @@ def spree_storefront_resource_url(resource, options = {})
       end
     end
 
+    # Helper method to determine the correct active option for navigation links
```

Comment:
@jawaria-sparksol, where is this method used? if it's needed please apply it to the default theme